esiclient/v1/cluster/cluster.py

```diff
@@ -234,64 +308,57 @@ def get_parser(self, prog_name):
         parser = super(Undeploy, self).get_parser(prog_name)
 
         parser.add_argument(
-            "cluster_config_file",
-            metavar="<cluster_config_file>",
-            help=_("File describing the cluster configuration"))
+            "cluster_uuid",
```

Comment:
Is cluster_uuid retrieved by listing clusters? Could you add the instructions in README file?

Reply:
Yep, it's already there in the `openstack esi cluster undeploy` documentation.